guest/api/types.go

```diff
@@ -97,3 +113,8 @@ type NodeInfo interface {
 
 	Node() proto.Node
 }
+
+// PostFilterPlugin use nodeToStatusMap
+type NodeToStatusMap interface {
+	NodeToStatusMap() map[string]int
+}
```

Comment:
```suggestion
// NodeToStatusMap contains which Node got which status during the scheduling cycle.
type NodeToStatusMap interface {
	// NodeToStatusMap returns an actual map\
	// which is keyed by the node name and valued by the status code.
	NodeToStatusMap() map[string]StatusCode
```

Let's use StatusCode, it tells the meaning of map value implicitly